packages/block-library/src/comments/edit/comments-inspector-controls.js

```diff
@@ -17,8 +24,21 @@ export default function CommentsInspectorControls( {
 			"The <aside> element should represent a portion of a document whose content is only indirectly related to the document's main content."
 		),
 	};
+
+	const enhancedSubmissionNotice = __(
+		'Enhanced submission might cause interactive blocks within the Comment Template to stop working. Disable it if you experience any issues.'
```

Comment:
> what kinds of interactive blocks might stop working?

This is tricky because the correct answer is "interactive blocks that are not using the Interactivity API", which is too technical and doesn't make sense to put in the UI. We're still trying to come up with ways to auto-detect those blocks, but we haven't come up with a 100% reliable technique yet. Once we do, we can remove that message.

Reply:
> we haven't come up with a 100% reliable technique yet

I had an idea to make this 100% reliable and remove the warning. I'll open a discussion to talk about it. _It also applies to the [Query block pagination](https://github.com/WordPress/gutenberg/pull/53812)._

Reply:
There you go:

- https://github.com/WordPress/gutenberg/discussions/54194

Any input would be highly appreciated 🙂